x/concentrated-liquidity/math/tick_test.go

```diff
@@ -54,169 +54,198 @@ func PriceToTickRoundDownSpacing(price osmomath.BigDec, tickSpacing uint64) (int
 			tickIndex:     -11999700,
-			expectedPrice: osmomath.MustNewDecFromStr("0.070003"),
+			expectedPrice: osmomath.MustNewBigDecFromStr("0.070003"),
 		},
 		"One hundred thousandth cent increments at the tenths place: 1": {
 			tickIndex:     -999800,
-			expectedPrice: osmomath.MustNewDecFromStr("0.90002"),
+			expectedPrice: osmomath.MustNewBigDecFromStr("0.90002"),
 		},
 		"One hundred thousandth cent increments at the tenths place: 2": {
 			tickIndex:     -999700,
-			expectedPrice: osmomath.MustNewDecFromStr("0.90003"),
+			expectedPrice: osmomath.MustNewBigDecFromStr("0.90003"),
 		},
 		"One ten thousandth cent increments at the ones place: 1": {
 			tickIndex:     1000000,
-			expectedPrice: osmomath.MustNewDecFromStr("2"),
+			expectedPrice: osmomath.MustNewBigDecFromStr("2"),
 		},
 		"One dollar increments at the ten thousands place: 2": {
 			tickIndex:     1000100,
-			expectedPrice: osmomath.MustNewDecFromStr("2.0001"),
+			expectedPrice: osmomath.MustNewBigDecFromStr("2.0001"),
 		},
 		"One thousandth cent increments at the tens place: 1": {
 			tickIndex:     9200100,
-			expectedPrice: osmomath.MustNewDecFromStr("12.001"),
+			expectedPrice: osmomath.MustNewBigDecFromStr("12.001"),
 		},
 		"One thousandth cent increments at the tens place: 2": {
 			tickIndex:     9200200,
-			expectedPrice: osmomath.MustNewDecFromStr("12.002"),
+			expectedPrice: osmomath.MustNewBigDecFromStr("12.002"),
 		},
 		"One cent increments at the hundreds place: 1": {
 			tickIndex:     18320100,
-			expectedPrice: osmomath.MustNewDecFromStr("132.01"),
+			expectedPrice: osmomath.MustNewBigDecFromStr("132.01"),
 		},
 		"One cent increments at the hundreds place: 2": {
 			tickIndex:     18320200,
-			expectedPrice: osmomath.MustNewDecFromStr("132.02"),
+			expectedPrice: osmomath.MustNewBigDecFromStr("132.02"),
 		},
 		"Ten cent increments at the thousands place: 1": {
 			tickIndex:     27732100,
-			expectedPrice: osmomath.MustNewDecFromStr("1732.10"),
+			expectedPrice: osmomath.MustNewBigDecFromStr("1732.10"),
 		},
 		"Ten cent increments at the thousands place: 2": {
 			tickIndex:     27732200,
-			expectedPrice: osmomath.MustNewDecFromStr("1732.20"),
+			expectedPrice: osmomath.MustNewBigDecFromStr("1732.20"),
 		},
 		"Dollar increments at the ten thousands place: 1": {
 			tickIndex:     36073200,
-			expectedPrice: osmomath.MustNewDecFromStr("10732"),
+			expectedPrice: osmomath.MustNewBigDecFromStr("10732"),
 		},
 		"Dollar increments at the ten thousands place: 2": {
 			tickIndex:     36073300,
-			expectedPrice: osmomath.MustNewDecFromStr("10733"),
+			expectedPrice: osmomath.MustNewBigDecFromStr("10733"),
 		},
 		"Max tick and min k": {
 			tickIndex:     342000000,
-			expectedPrice: types.MaxSpotPrice,
+			expectedPrice: types.MaxSpotPriceBigDec,
 		},
-		"Min tick and max k": {
-			tickIndex:     types.MinInitializedTick,
-			expectedPrice: types.MinSpotPrice,
+		"tickIndex is MinInitializedTickV1": {
```

Comment:
Can we also have tests for Max cases?

Reply:
Made issue since this is outside of scope of the changes made here and would cause conflicts with a PR sequence made off of the branch here:
https://github.com/osmosis-labs/osmosis/issues/6333